feat: Provide support for Azure AD Workload Identity (#263)
kedacore/charts: keda/README.md

| `serviceAccount.name`                                      | The name of the service account to use. If not set and create is true, a name is generated.      | `keda-operator` |
| `serviceAccount.annotations`                               | Annotations to add to the service account | `{}` |
| `podIdentity.activeDirectory.identity`                     | Identity in Azure Active Directory to use for Azure pod identity | `` |

> | `podIdentity.azureWorkload.clientId`                       | Azure Active Directory Client ID use for Azure Workload Identity. | `` |

Reviewer: ```suggestion
| `podIdentity.azureWorkload.clientId`                       | Id of Azure Active Directory client to use for authentication with for Azure Workload Identity. ([docs](https://keda.sh/docs/concepts/authentication/#azure-workload-identity)) | `` |
```

> | `podIdentity.azureWorkload.enabled`                        | Specifies whether Azure Workload Identity is to be enabled. | `false` |

Reviewer: ```suggestion
| `podIdentity.azureWorkload.enabled`                        | Specifies whether [Azure Workload Identity](https://azure.github.io/azure-workload-identity) is to be enabled or not. ([docs](https://keda.sh/docs/concepts/authentication/#azure-workload-identity)) | `false` |
```

> | `podIdentity.azureWorkload.tenantId`                       | Azure Active Directory Tenant ID use for Azure Workload Identity. | `` |

Reviewer: What about this?
```suggestion
| `podIdentity.azureWorkload.tenantId`                       | Id of Azure Active Directory tenant to use for authentication with Azure Workload Identity. ([docs](https://keda.sh/docs/concepts/authentication/#azure-workload-identity)) | `` |
```

> | `podIdentity.azureWorkload.tokenExpiration`                | Expiration duration (seconds) for projected service account tokens. | `3600` |

Reviewer: What about this?
```suggestion
| `podIdentity.azureWorkload.tokenExpiration`                | Duration in seconds to automatically expire tokens for the service account. ([docs](https://keda.sh/docs/concepts/authentication/#azure-workload-identity)) | `3600` |
```
